### PR TITLE
feat: make client id required

### PR DESCRIPTION
**Pull request by ieow**
added allow api test
fix allow api call
due to changes on https://github.com/torusresearch/torus-signer/pull/55

### CODE REVIEW

File: Tests/TorusUtilsTests/SapphireTest.swift [hunk 1]
```diff
@@ -336,4 +336,32 @@ final class SapphireTest: XCTestCase {
...
+        do {
+            let data = try await torus.retrieveShares(
+                endpoints: nodeDetails.getTorusNodeEndpoints(),
+                torusNodePubs: nodeDetails.getTorusNodePub(),
+                indexes: nodeDetails.getTorusIndexes(),
+                verifier: "w3a-auth0-demo",
+                verifierParams: verifierParams,
+                idToken: token
+            )
+            XCTAssert(false, "Should not pass")
```
Comment (metalurgical) [inline Tests/TorusUtilsTests/SapphireTest.swift:358]: Shouldn't this be success instead of failure?

Comment (ieow) [reply Tests/TorusUtilsTests/SapphireTest.swift:358]: It should fail because of invalid client id

**Comment by metalurgical** [comment]
Close #83

**Comment by metalurgical** [comment]
Close #82

**Review by metalurgical — APPROVED** [review]

**Pull request merged**